tests/test_warpedPsf.cc

```diff
@@ -247,24 +309,23 @@ BOOST_AUTO_TEST_CASE(warpedPsf)
     unwarped_psf->evalABC(a, b, c, q);
 
     Eigen::Matrix2d m0;
-    m0 << a, b,
-          b, c;
+    m0 << a, b, b, c;
 
     AffineTransform atr = lsst::afw::geom::linearizeTransform(*distortion->inverted(), p);
 
     Eigen::Matrix2d md;
-    md << atr.getLinear()[0], atr.getLinear()[2],
-          atr.getLinear()[1], atr.getLinear()[3];   // LinearTransform transposed index convention
+    md << atr.getLinear()[0], atr.getLinear()[2], atr.getLinear()[1],
```

Comment:
This seems like a regression in the formatting to me too.  Before it was clear that a 2x2 matrix was being processed.  Now that's less clear.

(Not trying to get into a format war here, but if there's a way to satisfy automated tools / standards and maintain useful information like "this is a 2x2 matrix", that'd be great.  If not, then carry on I guess.)